src/transformers/models/vivit/modeling_vivit.py

```diff
@@ -532,8 +532,17 @@ def forward(
 
 
         >>> def sample_frame_indices(clip_len, frame_sample_rate, seg_len):
+        ...     '''
+        ...     Sample frames from the video.
+        ...     Args:
+        ...         clip_len (`int`): Total number of frames to sample
+        ...         frame_sample_rate (`int`): Frame sample rate
+        ...         seg_len (`int`):  Number of frames in a sample segment
+        ...     Returns:
+        ...         result (`List[int]`): List of sampled video frame indices
+        ...     '''
```

Comment:
To match library doc string formatting 

```suggestion
        ...     """
        ...     Sample frames from the video.
        ...     Args:
        ...         clip_len (`int`): 
        ...             Number of frames to sample
        ...         frame_sample_rate (`int`):
        ...             Frame sample rate
        ...         seg_len (`int`):
        ...             Total number of frames in a sample segment
        ...    
        ...     Returns:
        ...         indices (`List[int]`): List of sampled video frame indices
        ...     """
```